drivers/timer/nrf_rtc_timer.c

```diff
@@ -210,8 +318,14 @@ int z_clock_driver_init(const struct device *device)
 	nrf_rtc_task_trigger(RTC, NRF_RTC_TASK_CLEAR);
 	nrf_rtc_task_trigger(RTC, NRF_RTC_TASK_START);
 
+	if (CONFIG_NRF_RTC_TIMER_USER_CHAN_COUNT) {
+		int_mask = BIT_MASK(EXT_CHAN_COUNT) << 1;
```

Comment:
For consistency, I think this should be `BIT_MASK(CHAN_COUNT)`, as the interrupt for the system clock is actually enabled.